src/graph/service/GraphService.cpp

```diff
@@ -114,6 +114,48 @@ void GraphService::signout(int64_t sessionId) {
   sessionManager_->removeSession(sessionId);
 }
 
+folly::Future<ExecutionResponse> GraphService::future_executeWithParameter(
+    int64_t sessionId,
+    const std::string& query,
+    const std::unordered_map<std::string, Value>& parameterMap) {
```

Comment:
Could we unify `future_executeWithParameter` and `future_execute`? They share almost the same code.

Reply:
After nebula-python's pr been merged.